src/OrleansServiceBus/Providers/Streams/EventHub/ICheckpointSettings.cs

```diff
@@ -0,0 +1,99 @@
+﻿
+using System;
+using System.Collections.Generic;
+using Orleans.Providers;
+
+namespace Orleans.ServiceBus.Providers
+{
+    public interface ICheckpointSettings
+    {
+        /// <summary>
+        /// Azure table storage data connections string
+        /// </summary>
+        string DataConnectionString { get; }
+
+        /// <summary>
+        /// Azure storage table name where the checkpoints will be stored
+        /// </summary>
+        string TableName { get; }
+
+        /// <summary>
+        /// How often to persist the checkpoints, if they've changed.
+        /// </summary>
+        TimeSpan PersistCadence { get; }
+
+        string DeploymentId { get; }
```

Comment:
what is exactly DeploymentId? Will it typically map to ServiceId? what should people use here? Is it related to a consumer group in event hub?